algorand/test/test.py

```diff
@@ -134,10 +134,13 @@ def getVAA(self, client, sender, sid, app):
         while True:
             nexttoken = ""
             while True:
-                response = self.myindexer.search_transactions( min_round=self.INDEXER_ROUND, note_prefix=self.NOTE_PREFIX, next_page=nexttoken)
+                response = self.myindexer.search_transactions( min_round=self.INDEXER_ROUND, next_page=nexttoken)
```

Comment:
tweaked this line for my own env, might not be what you want?

Reply:
It does not hurt since it is test net only.. just makes things slower